localstack/services/secretsmanager/provider.py

```diff
@@ -21,7 +21,12 @@
 from moto.secretsmanager.exceptions import (
     SecretHasNoValueException as MotoSecretHasNoValueException,
 )
-from moto.secretsmanager.exceptions import SecretNotFoundException as MotoSecretNotFoundException
+from moto.secretsmanager.exceptions import (
+    SecretNotFoundException as MotoSecretNotFoundException,
+)
+from moto.secretsmanager.exceptions import (
+    SecretStageVersionMismatchException as MotoSecretStageVersionMismatchException,
+)
```

Comment:
(non-blocking) Consider importing `moto.secretsmanager.exceptions` and using exceptions by attribute access